FragmentService/FragmentExtraContentInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\ArticleBundle\FragmentService;
+
+use Sonata\ArticleBundle\Model\FragmentInterface;
+
+/**
+ * @author Benoit Mazière <benoit.maziere@ekino.com>
+ */
+interface FragmentExtraContentInterface
```

Comment:
- It's already in the `FragmentService` namespace, no need to repeat `Fragment`
- It provides extra content

=> `ExtraContentProviderInterface`? WDYT @qsomazzi ?